tools/spell/README.md

```diff
@@ -1,15 +1,13 @@
 # How to create a lexicon.txt for a new library.
 
-1. In your GNU environment install programs: *spell* and *getopt*  
-   For Linux here are the commands to install these programs:  
+1. In your GNU environment install the *spell* program. Use the following command in Linux, to install the package:
    ```shell
```

Comment:
`getopt` was already part of my default Ubunutu 18.04 installation. 
The command `apt-get install getopt` is incorrect in finding the package

Reply:
Is this true about different linux distributions?

Reply:
Not sure about other distros. Will need to check it. However, `apt-get install getopt` command is incorrect...

Reply:
Debian family uses apt-get and Redhat family uses Yum.

Reply:
Yea. If it is incorrect, can we just remove it?Can we assume that the target system is Ubunutu 18.04?

Reply:
Found from this manual (http://manpages.ubuntu.com/manpages/trusty/man1/getopt.1.html) that `util-linux` is the kernel package from which **getopt** can be installed. (Should be part of default installation in Linux environments)

Have updated to provide the correct command for installation of the utility.